src/main/java/com/wynntils/modules/utilities/overlays/inventories/FavoriteItemsOverlay.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *  * Copyright © Wynntils - 2018 - 2021.
+ */
+
+package com.wynntils.modules.utilities.overlays.inventories;
+
+import com.wynntils.Reference;
+import com.wynntils.core.events.custom.GuiOverlapEvent;
+import com.wynntils.core.framework.interfaces.Listener;
+import com.wynntils.core.framework.rendering.ScreenRenderer;
+import com.wynntils.core.framework.rendering.textures.Textures;
+import com.wynntils.core.utils.StringUtils;
+import com.wynntils.webapi.WebManager;
+import com.wynntils.webapi.profiles.item.ItemProfile;
+
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.inventory.Slot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.fml.common.eventhandler.EventPriority;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import static net.minecraft.util.text.TextFormatting.*;
+
+public class FavoriteItemsOverlay implements Listener {
+
+    private static void renderFavorites(GuiContainer gui) {
+        if (!Reference.onWorld) return;
+
+        for (Slot s : gui.inventorySlots.inventorySlots) {
+            ItemStack stack = s.getStack();
+            if (stack.isEmpty() || !stack.hasDisplayName() || !stack.hasTagCompound()) continue;
+            NBTTagCompound nbt = stack.getTagCompound();
+            if (!nbt.getBoolean("wynntilsFavorite")) continue;
+
+            // draw star
+            ScreenRenderer.beginGL(0, 0);
+            GlStateManager.translate(0, 0, 300);
+
+            ScreenRenderer r = new ScreenRenderer();
+            RenderHelper.disableStandardItemLighting();
+            float scaleFactor = 0.5f;
+            ScreenRenderer.scale(scaleFactor);
```

Comment:
Declaring the variable here seems unnecessary? Probably could be
```suggestion
            ScreenRenderer.scale(0.5f);
```